aws/cloudformation/rdp_bastion.yaml

```
@@ -455,13 +481,88 @@ Resources:
           AssociatePublicIpAddress: false
           DeviceIndex: 0
           DeleteOnTermination: true
-          SubnetId: !Ref rCustomerVpcPrivateSubnet
+          SubnetId: !Ref rCustomerVpcPrivateSubnetA
       Tags:
         - Key: Name
           Value: !Join ["", [!Ref pTagprefix, "-bastion-instance"]]
         - Key: "redhattraining:course-sku"
           Value: CS220
 
+  # Security Group for rds
+  rSecurityGroupDatabase:
+    Type: AWS::EC2::SecurityGroup
+    DependsOn: rCustomerVpc
+    Properties:
+      GroupDescription: Enable mysql connectivity to Rosa cluster
+      GroupName: Database Security Group
+      VpcId: !Ref rCustomerVpc
+      Tags:
+        - Key: Name
+          Value: !Join ["", [!Ref pTagprefix, "-database-sg"]]
+        - Key: "redhattraining:course-sku"
+          Value: CS220
+      SecurityGroupIngress:
+        - CidrIp:
+            Fn::ImportValue: RosaVpcCidr
+          FromPort: 3306
+          ToPort: 3306
+          IpProtocol: tcp
+      SecurityGroupEgress:
+        - CidrIp:
+            Fn::ImportValue: RosaVpcCidr
+          FromPort: 1024
+          ToPort: 65535
+          IpProtocol: tcp
+
+
+  # Database subnet group
+  rRdsDBSubnetGroup:
+    Type: 'AWS::RDS::DBSubnetGroup'
+    Properties:
+      DBSubnetGroupName: testdbsubnetgroup
+      DBSubnetGroupDescription: custom db subnet group
+      SubnetIds:
+        - !Ref rCustomerVpcPrivateSubnetA
+        - !Ref rCustomerVpcPrivateSubnetB
+      Tags:
+        - Key: Name
+          Value: !Join ["", [!Ref pTagprefix, "-db-subnetgroup"]]
+        - Key: "redhattraining:course-sku"
+          Value: CS220
+
+  # RDS mysql instance. Application on Rosa will connect to it.
+  rRdsInstance:
+    Type: 'AWS::RDS::DBInstance'
+    Properties:
+      DBInstanceClass: db.t3.micro
+      MultiAZ: false
+      AvailabilityZone:
+        Fn::Select:
+          - 0
+          - Fn::GetAZs: ""
+      Engine: mysql
+      EngineVersion: 8.0.33
+      MasterUsername: admin
+      MasterUserPassword: redhat123
+      Port: '3306'
+      DBInstanceIdentifier: rds-mysql
+      AllocatedStorage: '20'
+      DBName: quotesdb
+      StorageEncrypted: false
+      StorageType: standard
+      BackupRetentionPeriod: 1
+      DeleteAutomatedBackups: true
+      DeletionProtection: false
+      DBSubnetGroupName: 
```

trailing whitespace:

```suggestion
      DBSubnetGroupName:
```